packages/server/graphql/types/RetroReflection.ts

```diff
@@ -45,6 +46,26 @@ const RetroReflection = new GraphQLObjectType<Reflection, GQLContext>({
       type: GraphQLID,
       resolve: resolveForSU('creatorId')
     },
+    creator: {
+      description: 'The user that created the reflection, only visible if anonymity is disabled',
+      type: User,
+      resolve: async ({creatorId, meetingId}, _args: unknown, {dataLoader}) => {
+        const meeting = await dataLoader.get('newMeetings').load(meetingId)
+
+        const {meetingType} = meeting
+
+        if (meetingType !== 'retrospective') {
+          return null
+        }
+
+        if (!meeting.disableAnonymity) {
```

Comment:
+1 `not disabled anonymity` means the same as anonymous, it just feels a bit strange (aha, my head reversed three times 😂), maybe give it a more reasonable name, depending on you.